src/data-browser/graphs/quarterly/index.jsx

```diff
@@ -21,12 +21,15 @@ export const QuarterlyGraphs = (props) => {
   const showGraphs = ![PATH_FILERS_INFO, PATH_FAQ].includes(location.pathname)
 
   return (
-    <div className="Graphs">
+    <div className='Graphs'>
       <HomeLink />
       <GraphsHeader overview={graphHeaderOverview} />
       <Error error={error} />
       <SectionSelector props={props} />
-      <div className="section-wrapper">
+      <div
+        className='section-wrapper'
+        style={location.pathname.includes('/faq') ? { marginTop: '-.6em' } : {}}
```

Comment:
Let's avoid inline styles where possible.  They can get messy as the number of required styles  grows. Since we're always applying this to the FAQ section, we could
- add a class `dynamic-renderer` to the wrapper div in `<DynamicRenderer />`
- Apply the margin via CSS. Selector: `.Graphs .dynamic-renderer`